elements/pfe-styles/src/pfe-typography-deprecated.scss

```diff
@@ -16,7 +16,7 @@
     @extend %pfe-text;
 }
 .pfe-text-sm {
-    @extend %pfe-title--md;
+    @extend %pfe-text-sm;
```

Comment:
Shouldn't this be `pfe-text--sm`?

Reply:
The new ones have two dashes, but this is the deprecated file & the old extends had only one dash